go-controller/pkg/ovn/namespace.go

```diff
@@ -331,6 +331,30 @@ func (oc *Controller) deleteNamespace(ns *kapi.Namespace) {
 		return
 	}
 	defer nsInfo.Unlock()
+
+	klog.V(5).Infof("Deleting Namespaces's NetworkPolicy entities")
```

Comment:
I suspect there is more cleanup that will need to be done for namespace cleanup, could you wrap this in a function like `(oc *Controller) cleanUpNsinfo()`

Reply:
ack... I agree, this was more of a prototype to make sure it stopped the race.  What else do you think needs to be cleaned up for NPs? I basically mimicked the functionality of `deleteNetworkPolicy` minus `oc.localPodDelDefaultDeny(policy, portInfo)` which based on my understanding shouldn't matter here since all the Pods in the namespace will be deleted

Reply:
fixed